src/Pyz/Yves/ShopUi/Theme/default/components/molecules/search-form/search-form.scss

```diff
@@ -17,11 +17,6 @@
                 position: relative;
                 transform: none;
                 transition: none;
-                z-index: $setting-zi-element;
-            }
-
-            .body-overlay--no-search & {
-                z-index: $setting-zi-search-suggestions;
```

Comment:
Default `$setting-zi-search-suggestions` not used anymore, only in the calculation, please assign this value to some element and calculate others from him.